src/validator.cpp

```diff
@@ -492,6 +492,69 @@ class Validator::ValidatorImpl: public LoggerImpl
      */
     void validateAndCleanMathCiCnNodes(XmlNodePtr &node, const ComponentPtr &component, const NameList &variableNames);
 
+    /**
+     * @brief Add a MathML-related issue.
+     *
+     * Add a MathML-related issue.
+     *
+     * @param description The description for the MathML-related issue.
+     * @param referenceRule The reference rule for the MathML-related issue.
+     * @param component The component where the MathML-related issue occurred.
+     */
+    void addMathMLIssue(const std::string &description,
```

Comment:
Random observation - not consistent in `MathML` vs `Mathml` :)

Reply:
Agreed! Changed to `addMathmlIssue`.

d70e227b